railties/test/application/rake_test.rb

```diff
@@ -127,6 +127,30 @@ def test_rake_routes_with_controller_environment
       assert_equal "Prefix Verb URI Pattern     Controller#Action\n  cart GET  /cart(.:format) cart#show\n", output
     end
 
+    def test_rake_routes_with_controller_environment
```

Comment:
This name doesn't match what you're testing - and there's a test just above it, which is called the same thing :smile: